front/lib/plans/usage/mau.ts

```diff
@@ -0,0 +1,90 @@
+import type { LightWorkspaceType, Result } from "@dust-tt/types";
+import { Err, Ok } from "@dust-tt/types";
+import { QueryTypes } from "sequelize";
+import type Stripe from "stripe";
+
+import { updateStripeActiveUsersForSubscriptionItem } from "@app/lib/plans/stripe";
+import type { MauReportUsageType } from "@app/lib/plans/usage/types";
+import { InvalidRecurringPriceError } from "@app/lib/plans/usage/types";
+import { getFrontReplicaDbConnection } from "@app/lib/resources/storage";
+
+export async function getActiveUsersInWorkspaceForPeriod({
```

Comment:
would name it `countMAUsForPeriod`
- return is  a number, get lets us believe we get a list of users
- InWorkspace can be implicit;
- MAU is more precise on what the function does

Reply:
Renamed in `getActiveUsersInWorkspaceForPeriod`.